index.html

```diff
@@ -12,6 +12,8 @@ <h1 class="hero-main-header small-12">Next Workshop</h1>
                     </div>
                     <h2 class="hero-main-header small-12">Upcoming Workshops</h2>
                     <div align="center">
+                    <p align="left" class="small-10 small-offset-1 medium-6 medium-offset-3">05-06 June in Brno, Elmi meeting -- <a href="{{ site.baseurl }}/events/omero-users-workshop-brno-june-2019.html" style="color:white">See program</a></p>
```

Comment:
Elmi -> ELMI

see for example 

![Screen Shot 2019-04-24 at 11 43 46](https://user-images.githubusercontent.com/2478303/56653831-7cfb3700-6686-11e9-9c13-a786be20b9f4.png)